package.json

```diff
@@ -22,5 +22,12 @@
   "homepage": "https://github.com/debitoor/find-dependencies",
   "dependencies": {
     "async": "^1.4.2"
```

Comment:
please fix version of dependency and install @debitoor/mocha-strict-dependencies